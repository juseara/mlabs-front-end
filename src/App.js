import React from 'react';
import GlobalStyle from './styles/global';
import Routes from './routes'
import { Header } from './components'

function App() {
  return (
    <>
      <Header />
      <main >
        <Routes />
      </main>
      <GlobalStyle/>
    </>
  );
}

export default App;
