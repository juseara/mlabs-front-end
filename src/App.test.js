import React from 'react';
import { shallow } from 'enzyme';
import App from './App';


describe('[Component] App',() =>{

  it('deve montar corretamente', () => {
      const wrapper = shallow(<App />)
      expect(wrapper).toMatchSnapshot();
  });

})
