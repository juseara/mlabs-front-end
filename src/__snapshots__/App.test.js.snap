// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] App deve montar corretamente 1`] = `ShallowWrapper {}`;
